### PR TITLE
curvefs/client: use separate thread to call SetDiskInitUsedBytes(#1842)

### PR DESCRIPTION
**Pull request by wuhongsong**
SetDiskInitUsedBytes may takes a long time, so use a separate thread to do this.
otherwise it will block FuseOpInit

signed-off-by: hzwuhongsong hzwuhongsong@corp.netease.com

### What problem does this PR solve?

Issue Number: #1842

### CODE REVIEW

**Comment by wuhongsong** [comment]
recheck

File: curvefs/src/client/s3/disk_cache_manager.cpp [hunk 1]
```diff
@@ -450,8 +450,11 @@ void DiskCacheManager::InitMetrics(const std::string &fsName) {
     cacheWrite_->InitMetrics(metric_);
     cacheRead_->InitMetrics(metric_);
     // this function move to here from init，
-    // Otherwise, you can't get the original metric
-    SetDiskInitUsedBytes();
+    // Otherwise, you can't get the original metric.
+    // SetDiskInitUsedBytes may takes a long time,
+    // so use a separate thread to do this.
+    Thread t(&DiskCacheManager::SetDiskInitUsedBytes, this);
+    t.detach();
```
Comment (xu-chaojie) [inline curvefs/src/client/s3/disk_cache_manager.cpp:457]: It is not a good idea to use a detach thread，if thead exec function SetDiskInitUsedBytes after DiskCacheManager destruct，it will cause an exception.

Comment (ilixiaocui) [reply curvefs/src/client/s3/disk_cache_manager.cpp:457]: +1

**Review by xu-chaojie — APPROVED** [review]

File: curvefs/src/client/s3/disk_cache_manager.cpp [hunk 2]
```diff
@@ -450,8 +457,11 @@ void DiskCacheManager::InitMetrics(const std::string &fsName) {
     cacheWrite_->InitMetrics(metric_);
     cacheRead_->InitMetrics(metric_);
     // this function move to here from init，
-    // Otherwise, you can't get the original metric
-    SetDiskInitUsedBytes();
+    // Otherwise, you can't get the original metric.
+    // SetDiskInitUsedBytes may takes a long time,
+    // so use a separate thread to do this.
+    diskInitThread_ = std::thread(
```
Comment (xu-chaojie) [inline curvefs/src/client/s3/disk_cache_manager.cpp:463]: use curve::common::Thread

Comment (wuhongsong) [reply curvefs/src/client/s3/disk_cache_manager.cpp]: fix

File: curvefs/src/client/fuse_client.h [hunk 3]
```diff
@@ -326,18 +326,18 @@ class FuseClient {
...
+            VLOG(3) << "del lookpath end";
             return;
         }
         splitPath->push_back(pToken);
         while (true) {
             pToken = strtok_r(NULL, const_cast<char*>(
               delimiter.c_str()), &pSave);
             if (nullptr == pToken) {
-                VLOG(3) << "whsdel lookpath end";
+                VLOG(3) << "del lookpath end";
```
Comment (ilixiaocui) [inline curvefs/src/client/fuse_client.h:337]: VLOG(6) ?

**Comment by wuhongsong** [comment]
recheck

**Comment by wuhongsong** [comment]
recheck

**Review by ilixiaocui — APPROVED** [review]
